App/App.pro

```diff
@@ -79,7 +79,7 @@ android: {
 # Remember to actually run it: (custom step in QtCreator, "make assetsrcc" from commandline)
 # run from ${buildDir}/App
 # /usr/bin/make assetsrcc
-include(assets.pri)
+//include(assets.pri)
```

Comment:
Please leave the building process as is.
The game runs on various platforms where the benefit of building this way outweighs the negatives (the extra make step) :)